installation-bundle/src/InstallTool.php

```diff
@@ -384,14 +384,35 @@ public function persistAdminUser(string $username, string $name, string $email,
             '=' => '&#61;',
         ];
 
-        $statement->executeStatement([
```

Comment:
`$statement = $this->connection->prepare("…` on line 355 needs to be removed I think.

Reply:
Fixed in e7ef92549dcec88463628f5bd274df6c13879fa9.